features/anchor-positioning.yml.dist

```diff
@@ -91,7 +91,6 @@ compat_features:
   - css.properties.bottom.anchor
   - css.properties.height.anchor-size
   - css.properties.inline-size.anchor-size
-  - css.properties.inset.anchor
```

Comment:
It would be nice to have but https://github.com/web-platform-dx/web-features/pull/1231 will eventually make this sort order irrelevant, right? Or at least it's a problem for that code?

Reply:
Right... I was thinking https://github.com/web-platform-dx/web-features/pull/1231 would only group things but not change the order, but actually it will be quite natural only order the keys that we're going to write to dist, and approach it more as "nice visual display order" and not "canonicalize for isDeepStrictEqual". Details TBD, but I think you're right it's fine to solve this problem there, if the order feels ugly after breaking up into blocks.